doc: added Updating an application section to README (#111)
HIP-infrastructure/app-in-browser: README.md

```bash
./buildapp.py [app_image_name]
```

> ## Updating an application

Reviewer: ```suggestion

## Updating an application
```

Changes and testing for development of an application must be done on the dev servers.

When the changes are ready, to be tested, commit them to the dev branch

If you need to build the containers on the machine, you will need to:

Permit the networks to allow access to the docker registry
```

> app-in-browser/scripts/permitnetwork.sh

Reviewer: ```suggestion
./scripts/permitnetwork.sh
```

```

> - Build the containers you want to build, using the scripts in scripts/ 
> - Restrict the networks again

Reviewer: ```suggestion

Build the containers you want to build, using the scripts in scripts/ 

Restrict the networks again
```

```

> app-in-browser/scripts/restrictnetwork.sh

Reviewer: ```suggestion
./scripts/restrictnetwork.sh
```

./buildapp.py [app_image_name]
```

## Updating an application

Changes and testing for development of an application must be done on the dev servers.

When the changes are ready, to be tested, commit them to the dev branch

If you need to build the containers on the machine, you will need to:

Permit the networks to allow access to the docker registry
```
./scripts/permitnetwork.sh
```

> Build the containers you want to build, using the scripts in scripts/ 

Reviewer 2: ```suggestion
Build the containers you want to build (ensure they are present in the the hip.yml)
`./scripts/buildapp.py myappname`
```
